views/utilities.js

```diff
@@ -227,6 +227,39 @@ module.exports  = {
         return requestData;
     },
 
+    // Searches for any previous request to see if this is a duplicate
+    // Returns null if not found, otherwise returns first record
+    findPriorRequest: function(requestData) {
+      // Replacing non-numeric characters with Postgres match operator, adding 
+      // one to the beginning to accommodate country codes
+      var phoneQuery = "%" + requestData.phone.replace(/[^0-9]/g, "%");
+
+      return db.Request.findOne({
```

Comment:
Should this be just a regular `find`?  What if there's more than one duplicate?  (I don't think I mentioned this case in the issue, but...maybe we take the most recent duplicate?)

Reply:
I think it'd have to be the most recent, even if there are more than one, so I added an ordering by `createdAt` in the latest commit

Reply:
+1